Code/GraphMol/MolPickler.cpp

```diff
@@ -867,6 +868,15 @@ void MolPickler::_pickle(const ROMol *mol, std::ostream &ss,
     _pickleSSSR<T>(ss, ringInfo, atomIdxMap);
   }
 
+  // Write Stereo Groups
```

Comment:
Since you're adding something new to the pickle, the version number needs to be changed. 
Please bump to 10.0 since the new pickles that contain stereo info will not be readable by old versions of the code.